### PR TITLE
Model Comparison (object detection and instance segmentation)

### PR DESCRIPTION
**Pull request by almazgimaev**

### CODE REVIEW

File: supervisely/nn/benchmark/coco_utils/utils.py [hunk 1]
```diff
@@ -1,15 +1,15 @@
 from supervisely.io.json import load_json_file
 
 
-def read_coco_datasets(cocoGt_json, cocoDt_json):
+def read_coco_datasets(cocoGt, cocoDt):
```
Comment (almazgimaev) [inline supervisely/nn/benchmark/coco_utils/utils.py:4]: need check

File: supervisely/nn/benchmark/evaluation/coco/metric_provider.py [hunk 2]
```diff
@@ -146,8 +148,8 @@ def json_metrics(self):
         ap_custom_by_class = dict(zip(self.cat_names, ap_custom_by_class))
         return {
             "mAP": base["mAP"],
-            "AP50": self.coco_metrics["AP50"],
-            "AP75": self.coco_metrics["AP75"],
+            "AP50": self.coco_metrics.get("AP50", "-"),
```
Comment (max-unfinity) [inline supervisely/nn/benchmark/evaluation/coco/metric_provider.py:151]: None is better here. We can replace None to "-" later, in visualizer

Comment (almazgimaev) [reply supervisely/nn/benchmark/evaluation/coco/metric_provider.py]: fixed

File: supervisely/nn/benchmark/visualization/evaluation_result.py [hunk 3]
```diff
@@ -0,0 +1,267 @@
+import pickle
```
Comment (max-unfinity) [inline supervisely/nn/benchmark/visualization/evaluation_result.py:1]: move to comparison folder

File: supervisely/nn/benchmark/comparison/visualization/visualizer.py [hunk 4]
```diff
@@ -0,0 +1,302 @@
...
+from supervisely.nn.benchmark.visualization.renderer import Renderer
+from supervisely.nn.benchmark.visualization.widgets import (
+    ContainerWidget,
+    GalleryWidget,
+    MarkdownWidget,
+    SidebarWidget,
+)
+
+
+class ComparisonVisualizer:
```
Comment (max-unfinity) [inline supervisely/nn/benchmark/comparison/visualization/visualizer.py:25]: rename to DetectionComparisonVisualizer

File: supervisely/nn/benchmark/visualization/evaluation_result.py (hunk 3 above)
Comment (max-unfinity) [reply supervisely/nn/benchmark/visualization/evaluation_result.py:1]: for later refactoring

**Review by max-unfinity — APPROVED** [review]

**Pull request merged**